db/shadower.go

```diff
@@ -123,6 +123,13 @@ func (s *Shadower) pullDocument(key string, value []byte, isDeletion bool, cas u
 		body["_rev"] = newRev
 		if doc.History[newRev] == nil {
 			// It's a new rev, so add it to the history:
+			if parentRev != "" && !doc.History.contains(parentRev) {
+				// parent rev does not exist in the doc history
+				// set parentRev to "", this will create a  new conflicting
+				//branch in the revtree
+				base.Warn("Adding revision as conflict branch, parent id %q is missing", parentRev)
```

Comment:
Can you add something to this warning message to indicate that this is happening during bucket shadowing pull?